### PR TITLE
Show / Download Exported recordings UI

### PR DESCRIPTION
**Pull request by NickM-27**
Adds the ability to see and download exported recordings. It also shows an activity indicator for exports that are in progress

Screen Shot 2023-07-15 at 07 38 26 AM

### CODE REVIEW

File: web/src/routes/Export.jsx [hunk 1]
```diff
@@ -22,6 +27,10 @@ export default function Export() {
   const [endDate, setEndDate] = useState(localISODate);
   const [endTime, setEndTime] = useState('23:59');
 
+  useEffect(() => {
```
Comment (blakeblackshear) [inline web/src/routes/Export.jsx:30]: Why not use swr here?

Comment (NickM-27) [reply web/src/routes/Export.jsx:30]: Is there a way to make swr not use the configured base path? The default base path includes /api

Comment (NickM-27) [reply web/src/routes/Export.jsx:30]: Actually I think I see what you mean, I'll fix that

**Review by blakeblackshear — APPROVED** [review]

**Comment by T3chArmy** [comment]
@NickM-27 any chance of you adding the ability to delete these exports? That would also be great!

**Comment by kdill00** [comment]
@NickM-27 Does adding -tag:v hvc1 for h265 cause issues on android devices?  I don't have one to test.  iOS needs this still for playback in safari.  I have just been editing the ffmpeg_presets.py file for timelapse and h265 record.

**Comment by NickM-27** [comment]
@kdill00 I don't use that and I have no issues playing back h265 in safari on my iPhone

**Comment by kdill00** [comment]
@NickM-27 okay, I will look at what im doing again. I dont know why my setup needs it.  If I dont add the tag, I can only download the clips into file browser and open with vlc.  I do not use go2rtc to injest the stream though, only to convert it to h264 for live playback when needed.  I use another separate service with to duplicate my streams which might be my issue.

**Pull request merged**